js/widgets/meterwidget.js

```diff
@@ -25,7 +25,7 @@ function MeterWidget() {
 
         let w = window.innerWidth;
```

Comment:
Please use const over let for variables which are not being re-assigned. Do this across the file. Pull the latest changes then try linting the file with ES-Lint to find such issues. We have enabled prefer-const rule in ES-Lint recently.